src/PokemonSuper.java

```diff
@@ -0,0 +1,140 @@
+package src;
+
+public abstract class PokemonSuper {
+    final String name;
+    String nameOfTrainer;
+    int level;
+    private int healthPoints;
+    private int experiencePoints;
+
+    String pokemonTalk;
```

Comment:
Waarom zijn deze niet private? Voor public (of package private) variabelen hoef je geen getters/setters te maken.